### PR TITLE
Comment out duplicate keys

### PR DESCRIPTION
**Pull request by MuXiu1997**
Comment out duplicate keys

reason: In some scripts used for localization, the key is expected to be unique

- context: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10419

### CODE REVIEW

**Review by draknyte1 — COMMENTED** [review]
I don’t see what you’re doing here beyond commenting out some keys.. the keys are case sensitive, so some duplicates exist value wise, but have unique keys case wise.

**Comment by MuXiu1997** [comment]
> I don’t see what you’re doing here beyond commenting out some keys.. the keys are case sensitive, so some duplicates exist value wise, but have unique keys case wise.

For example:
https://github.com/GTNewHorizons/GTplusplus/blob/3ead7cae28a1ba038345c09af14e3c3c18f3f259/src/main/resources/assets/miscutils/lang/en_US.lang#L3112-L3118

https://github.com/GTNewHorizons/GTplusplus/blob/3ead7cae28a1ba038345c09af14e3c3c18f3f259/src/main/resources/assets/miscutils/lang/en_US.lang#L3141-L3147

**Comment by MuXiu1997** [comment]
Not sure what value you really want to use, but apparently the key is duplicated and an override will occur

File: src/main/resources/assets/miscutils/lang/en_US.lang [hunk 1]
```diff
@@ -55,11 +55,11 @@ item.MiscUtils.material.dustCryotheum.name=Cryotheum Dust
 item.MiscUtils.material.rodBlizz.name=Blizz Rod
 item.MiscUtils.material.dustBlizz.name=Blizz Powder
 
-//============= 'miscutils' is Case Sensitive Here, unsure why
+//------------- 'miscutils' is Case Sensitive Here, unsure why
```
Comment (MuXiu1997) [inline src/main/resources/assets/miscutils/lang/en_US.lang:58]:
```java
public static HashMap<String,String> parseLangFile(InputStream inputstream)
    {
        HashMap<String,String> table = Maps.newHashMap();
        try
        {
            Iterator iterator = IOUtils.readLines(inputstream, Charsets.UTF_8).iterator();

            while (iterator.hasNext())
            {
                String s = (String)iterator.next();

                if (!s.isEmpty() && s.charAt(0) != 35)
                {
                    String[] astring = (String[])Iterables.toArray(equalSignSplitter.split(s), String.class);

                    if (astring != null && astring.length == 2)
                    {
                        String s1 = astring[0];
                        String s2 = numericVariablePattern.matcher(astring[1]).replaceAll("%$1s");
                        table.put(s1, s2);
                    }
                }
            }

        }
        catch (Exception ioexception)
        {
            ;
        }
        return table;
    }
```
`//` is not a comment symbol

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**